frontend/src/metabase/query_builder/components/view/sidebars/QuestionDataSource.styled.jsx

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
```

Comment:
@akiselev in newer code we agreed to create folders for components and their styles like:
/QuestionDataSource
    QuestionDataSource.tsx
    QuestionDataSource.styled.tsx
    index.ts